### DISCUSSION TITLE
blog/2022/11/29/conditional-marginal-marginaleffects/

### DISCUSSION THREAD

**Discussion by giscus** [andrewheiss/ath-quarto#52] [Blog comments]
# blog/2022/11/29/conditional-marginal-marginaleffects/

Use the {marginaleffects} package to calculate tricky and nuanced marginal and conditional effects in generalized linear mixed models

https://www.andrewheiss.com/blog/2022/11/29/conditional-marginal-marginaleffects/

**Comment by dgalgom** [comment]
Very nice and helpful vignette indeed! There are a lot of resources that used "conditional" and "marginal" in an interchangeably way, and this sheds light into it. Just a comment, in the "ATE for a typical cluster" section within Conditional effects, you wrote `marginaleffects::predictions` where would be `marginaleffects::comparisons` 
Thank you so much for your interesting and neat explanations!

**Reply by andrewheiss** [reply]
Ah, good catch! It's fixed now—thanks!

**Comment by rduesing** [comment]
Hi, many thanks for your work. Can you explain why you used the variance instead of the standard deviation sigma_0 ?
ggplot() +
  stat_function(fun = ~dnorm(., mean = B0, sd = sigma_0^2),
                geom = "area", fill = clrs[4])
